src/logger.interfaces.ts

```diff
@@ -12,6 +12,7 @@ export interface LoggerConfig {
   excludeHeaders?: string[];
   reqIdHeader?: string;
   genReqId?: (r: Request) => string;
+  excludeReqPath?: string;
```

Comment:
I suggest use array for extensibility

```
  excludeReqPaths?: string[];
```

Reply:
I was thinking about a regexp as an enhancement, but I will leave it until the real case appears.
At the moment, the health check API is the only case we need to skip the logging.